build.gradle

```diff
@@ -20,6 +20,7 @@ apply plugin: 'com.github.ben-manes.versions'
 apply plugin: 'com.gradle.plugin-publish'
 apply plugin: 'com.vanniktech.android.junit.jacoco'
 apply plugin: "com.vanniktech.maven.publish"
+apply plugin: 'maven-publish'
```

Comment:
Remove this please. You can install the plugin locally via `installArchives` task